docs/INTRODUCTION.md

```diff
@@ -1,63 +1,57 @@
+## Basic Concepts
 
-``Sections`` are rectangular regions within an ``oveCanvas``. Each ``section`` runs a single OVE ``app``. ``Sections`` may span multiple clients and can overlap.
+An OVE `server` installation supports multiple `spaces`. A `space` is a collection of monitors, which may be attached to different computers, that together form a single display. OVE is designed to be used in Large High Resolution Display environments (such as a [Data Observatory](https://www.imperial.ac.uk/data-science/data-observatory/) that is 2.53m x 6.00m in size and 30,720 x 4,320 in resolution, consisting of 64 monitors). But, OVE is also suitable for use on much smaller displays.
 
-An ``app`` displays something on the display. The official **core** ``apps`` each provide a way to display a distinct form of commonly used content.
-They include an ``HTML app`` that can be used to display any visualization that run as web apps (subject to the limitations described in the [list of pitfalls to avoid](./PITFALLS.md)). 
+In each space, OVE runs within a number of `clients`. An OVE `client` is a browser window and typically runs full-screen on a single `screen`. A `screen` can be a single monitor or can span multiple monitors if they were all attached to a single computer. The arrangement of `clients` is described in the [`Spaces.json`](https://github.com/dsi-icl/ove/blob/master/packages/ove-core/src/client/Spaces.json) file.
 
+An OVE `application` (or `app`) includes server-side and client-side components. Each [OVE App](https://github.com/ove/ove-apps/packages) confirm to a [common structure](./APP_DEVELOPMENT.md/#application-structure). Each of the [OVE Apps](https://github.com/ove/ove-apps) provide a way to display a distinct form of commonly used content (subject to the limitations described in the [list of pitfalls to avoid](./PITFALLS.md)).
 
-![](images/concepts.svg)
+Each individual `instance` of an OVE `app` is designed to run within its own `section`. `Sections` may span multiple `clients` and can overlap. `Sections` are rectangular regions within an `oveCanvas`. Each `section` has its own `geometry` which is a combination of its `height`, `width`, and 2D coordinates (`x`, `y`). The `geometry` describes the region on which a `section` is deployed on an `oveCanvas`. An `oveCanvas` can also have `groups` of one or more `sections` and `groups`.
 
+Each individual OVE `app` can have its own `configuration` (or `config`) defined within a [config.json](./APP_DEVELOPMENT.md/#application-structure). The `config` is used to define named `states` other app-specific configuration, which are common to all app `instances`. Each individual `instance` of an OVE `app` can have its own `state` that can be accessed by providing the `id` of the `section` in which the `app` is deployed.
```

Comment:
"can have" -> "has"

Reply:
This is accurate: config.json is optional. See https://github.com/ove/ove/blob/master/packages/ove-lib-appbase/src/index.js#L27.